src/appleseed.shaders/src/maya/as_maya_anisotropic.osl

```diff
@@ -269,7 +269,7 @@ shader as_maya_anisotropic
                 );
         }
 
-        if (transparency)
+        if (max(transparency) > 0.0)
```

Comment:
There are two reasons. The most important is that maybe this is triggering a bug in OSL.
I was having false positives regarding the transparent() closure usage when using if (color) {}.
The second one is a style issue. if (color) {} implies a conversion from a color to a boolean and 
I think that's not a natural conversion. I find it strange.